@here/harp-materials/lib/SolidLineMaterial.ts

```diff
@@ -48,20 +49,20 @@ varying vec3 vColor;
 #include <extrude_line_vert_func>
 
 void main() {
-    vLinewidth = lineWidth;
+    vResultLineWidth = lineWidth + outlineWidth * 2.0;
```

Comment:
1. This is actually `lineExtrusionFactor`. (Which is apparently half actual lineWidth)

2.
Why `* 2 .0` here ?

At first look they cancel out with code in `TileGeometryCreator`.

Reply:
@zbigg `* 2.0` is b/c we have oulines from both sides (right, left) and outlineWidth is width of one of them

Reply:
@sasha240100

> outlineWidth is width of one of them

Yes, i understand, but ... why you divide it at start ?

So, you shouldn't multiply(here) and divide (there) at all.

Reply:
Ah, @zbigg got what you meant. Now I noticed that it is related to that `0.5` multiplication - fixed, will push a fix